src/Sylius/Bundle/ApiBundle/Resources/config/app/config.yaml

```
@@ -16,6 +16,7 @@ parameters:
         - "%sylius.security.new_api_route%/admin/promotion-actions/{id}"
         - "%sylius.security.new_api_route%/admin/promotion-rules/{id}"
         - "%sylius.security.new_api_route%/admin/shop-users/{id}"
+        - "%sylius.security.new_api_route%/admin/shipping-method-rules/{id}"
```

```suggestion
        - "%sylius.security.new_api_route%/admin/shipping-method-rules/{id}"
        - "%sylius.security.new_api_route%/admin/shop-users/{id}"
```
It would be nice to keep alphabetical order